src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -1,139 +1,197 @@
-import {
-  ClinicalEntity,
-  Donor,
-  ClinicalInfo,
-  ClinicalInfoStats,
-  AggregateClinicalInfoStats,
-} from '../../../src/clinical/clinical-entities';
-import { ClinicalEntitySchemaNames } from '../submission-entities';
-import { isNotAbsent, isEmpty, notEmpty } from '../../../src/utils';
-
-import * as schemaManager from '../schema/schema-manager';
-import { getClinicalObjectsFromDonor } from './submission-to-clinical';
-import { SchemasDictionary } from '../../../src/lectern-client/schema-entities';
+import { Donor } from '../../../src/clinical/clinical-entities';
```

Comment:
This file is really the bulk of the changes. I don't think its very clear to understand by looking at the diffs, so it might be easier to just see this file on its own.